app/models/course/mail_template.rb

```diff
@@ -0,0 +1,27 @@
+class Course::MailTemplate < ActiveRecord::Base
+  stampable
+  belongs_to :course
+  has_one :mail_sign_off, through: :course
+  # This is a list of mailer actions that allows customization.
+  enum action: { invitation: 0, announcement: 1 }
+
+  # Returns the content of customized sign off if it exists
+  def sign_off
+    if mail_sign_off
```

Comment:
Use a guard clause instead of wrapping the code inside a conditional expression.<br>Favor modifier `if` usage when having a single-line body. Another good alternative is the usage of control flow `&&`/`||`.